Jenkinsfile

```diff
@@ -9,7 +9,7 @@ pipeline {
   stages {
     stage('Test 2.2') {
       environment {
-        RUBY_VERSION = '2.2'
+        RUBY_VERSION = '2.2.0'
```

Comment:
instead of 0s - often buggy releases - we should use the latest `y` versions, for ruby versions `2.x.y`. This ensures that we're running against recent Rubies

Reply:
Changed to most recent `2.x` version.